internal/config/config.go

```diff
@@ -376,8 +389,15 @@ func GetKnownKeys() []string {
 
 // configPrefix is the main config structure passed to plugins, and used for root to wrap viper
 type configPrefix struct {
-	prefix string
-	keys   map[string]bool
+	prefix   string
+	keys     map[string]bool
+	defaults map[string][]interface{}
```

Comment:
Is this ever used? Seems like `defaults` are only used on `configPrefixArray` types and not `configPrefix` types.